code/modules/mob/living/carbon/human/species_types/android.dm

```diff
@@ -22,6 +22,7 @@
 		TRAIT_RESISTLOWPRESSURE,
 		TRAIT_TOXIMMUNE,
 		TRAIT_NOCRITDAMAGE,
+		TRAIT_HYPOSPRAYIMMUNE,
```

Comment:
alphabetize this list please. there's a few extensions for VSCode that will do this for you.